src/Tizen.NUI/src/public/VisualConstants.cs

```diff
@@ -148,7 +148,15 @@ public enum FittingModeType
         /// <summary>
         /// Image rows: Limit loaded image resolution to row height using the FitHeight mode.
         /// </summary>
-        FitHeight
+        FitHeight,
```

Comment:
Hello, 
This seems to be required ACR. (adding API)
because FittingModeType is public API.
could you check again?